src/Services/UserService.php

```diff
@@ -191,4 +191,12 @@ public function getAll($search = array(), $isAndCondition = true)
 
         return $results;
     }
+
+    /**
+     * @return array id of User
+     */
+    public function getIdByUsername($username)
+    {
+        return sqlQuery("SELECT `id` FROM `users` WHERE `username` = ?", [$username]);
```

Comment:
1. Would put a BINARY in front of `username`
2. Would return the id if it exists, and false otherwise

Reply:
Would also fix the getUserByUsername() function by putting BINARY in front of `username`